lightningd/chaintopology.c

```diff
@@ -474,7 +474,7 @@ u32 feerate_from_style(u32 feerate, enum feerate_style style)
 	case FEERATE_PER_KBYTE:
 		/* Everyone uses satoshi per kbyte, but we use satoshi per ksipa
 		 * (don't round down to zero though)! */
-		return (feerate + 3) / 4;
+		return feerate / 4 + 3;
```

Comment:
Would this lead to us computing a higher feerate than "Everyone" later on?

Reply:
I hesitated to use a "return FEERATE_FLOOR if < FEERATE_FLOOR ", but I think we were actually computing *lower* feerate than bitcoind before this.

Reply:
Okay.  Will ACK this.  Unless you want to change the second commit message.

Reply:
Will need to fixup the tests, so will amend the second commit message while at it.